docs/application/native/guides/app-management/ui-app/TOC.md

```diff
@@ -0,0 +1,12 @@
+# EFL Applications
+## [Overview](/application/native/guides/app-management/ui-app/efl-app.md)
+## [Basic UI Application](/application/native/guides/app-management/ui-app/efl-ui-app.md)
+## [Watch Application](/application/native/guides/app-management/ui-app/watch-app.md)
```

Comment:
This link is broken. There are no `watch-app.md` and `widget-app.md` file under `ui-app` folder.